### PR TITLE
Allow developers to override endpoints and return all the endpoints through the `getServiceEndpoints` method

### LINKED ISSUE #79: Allow developers to pass custom endpoints
The SDK already allows developers to pass custom endpoints. However, these endpoints are used only if the request to the well-known endpoint fails. The SDK should allow a developer should be able to choose the precedence, i.e., the developer should be able to use the custom endpoint instead of the ones returned by the well-known endpoint.

### LINKED ISSUE #80: Return the whole response from the request to the well-known endpoint
**Is your suggestion related to an experience ? Please describe.**
Only a selected few endpoints are returned by the `getServiceEndpoints` method. Instead, the method should return the whole response returned by the request to the well-known endpoint.

### PR DESCRIPTION
**Pull request by thivi**

### CODE REVIEW

**Review by DimalChandrasiri — APPROVED** [review]

File: packages/oidc-js/src/models/client.ts [hunk 1]
```diff
@@ -34,7 +34,9 @@ interface BaseConfigInterface {
     responseMode?: ResponseMode;
     scope?: string[];
     serverOrigin: string;
-    endpoints?: ServiceResourcesType;
+    endpoints?: OIDCEndpoints;
+    overrideWellEndpointConfig?: boolean;
```
Comment (brionmario) [inline packages/oidc-js/src/models/client.ts:38]: Shall we rename this to `overrideWellKnownEndpointConfig`?

May be we can do this in a separate PR.

Comment (brionmario) [reply packages/oidc-js/src/models/client.ts:38]: @thivi

Comment (thivi) [reply packages/oidc-js/src/models/client.ts:38]: Tracked here: https://github.com/asgardio/asgardio-js-oidc-sdk/issues/59
